ce/tests/test_api.py

```diff
@@ -32,6 +35,23 @@ def test_api_endpoints_are_callable(test_client, cleandb, endpoint, query_params
     url = '/api/' + endpoint
     response = test_client.get(url, query_string=query_params)
     assert response.status_code == 200
+    assert response.cache_control.public == True
+    assert response.cache_control.max_age > 0
```

Comment:
Do you want to be a bit more specific about `max_age`?

Reply:
Meh. We only set it in one and only one place, and the resolution of caching age is so coarse as to be trivial. I'm really only concerned that it got properly set to be anything aside from zero.